lib/google_crawler/keywords/keyword_importer.ex

```diff
@@ -0,0 +1,44 @@
+defmodule GoogleCrawler.Keywords.KeywordImporter do
+  @file_content_type "text/csv"
+  @keyword_separator ?,
+
+  alias GoogleCrawler.{Keywords, Repo}
+  alias GoogleCrawler.Keywords.KeywordImportError
+
+  def import(%{content_type: @file_content_type, path: keyword_path}, user_id) do
+    csv_stream = decode_csv(keyword_path)
+
+    Repo.transaction(fn -> do_import(csv_stream, user_id) end)
+  end
+
+  def import(_, _), do: {:error, "Invalid file type."}
+
+  defp decode_csv(keyword_path) do
+    keyword_path
+    |> File.stream!()
+    |> CSV.decode(separator: @keyword_separator)
```

Comment:
On `CSV.decode` it says it defaults to `?,` already
https://hexdocs.pm/csv/CSV.html#decode/2

But I'm a bit confused as to why we're declaring it here, are we planning to change the separator depending on the CSV? 🤔

Reply:
Removed the separator option and the separator always be `,` 🙌